Scaffold-Stark/scaffold-stark-2
# Implement separate Read and Write tabs in Debug Contracts

# Implement separate Read and Write tabs in Debug Contracts

Fixes #281

## Types of change


## Comments (optional)

This PR implements separate Read and Write tabs in the Debug Contracts page, addressing issue #281. The changes improve the user interface and organization of contract interactions.

### Changes made:
- Modified ContractUI component to use DaisyUI tabs
- Separated Read and Write methods into distinct tabs
- Improved UI layout for better user experience and clarity

### How to test:
1. Navigate to the Debug Contracts page
2. Select a contract
3. Verify that Read and Write methods are in separate tabs
4. Test switching between tabs and ensure correct content is displayed

### Additional notes:
- Maintained existing code structure as much as possible
- Used DaisyUI components for consistent styling with the rest of the application
